src/python/pants/option/options.py

```diff
@@ -12,7 +12,7 @@
 from pants.base.build_environment import get_buildroot
 from pants.base.deprecated import warn_or_error
 from pants.option.arg_splitter import ArgSplitter
-from pants.option.config import Config
+from pants.option.config import ConfigSource
```

Comment:
We no longer parse config in Python, so we only traffic in ConfigSource, which gets passed to Rust. 

A followup change will delete the Python Config class and its tests, but this change was already gnarly enough without adding that in.